Gruntfile.js

```diff
@@ -143,31 +141,54 @@ module.exports = function (grunt) {
 
   grunt.loadNpmTasks('grunt-eslint');
   grunt.loadNpmTasks('grunt-karma');
-  grunt.loadNpmTasks('grunt-browserify');
   grunt.loadNpmTasks('grunt-contrib-uglify');
 
   grunt.registerTask('test', [
     'eslint',
-    'browserify',
+    'compatibilityWithWebpackProd',
+    'buildWebpack',
```

Comment:
This verifies that running default UglifyJS (invoked on **webpack -p**) won't break the dist